### PR TITLE
Homework 3 Arrays ang Strings

### PR DESCRIPTION
**Pull request by AntTurieva**
Homework 3 Arrays ang Strings

### CODE REVIEW

File: src/main/java/hw_3/Task1.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            StringBuilder result = new StringBuilder();
+
+            for (int i = 0; i < arr.length; i++) {
+                result.append(arr[i]);
+                if (i < arr.length - 1) {
+                    result.append(";");
+                }
+            }
+
+            System.out.println(result.toString());
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: .toString() can be removed

File: src/main/java/hw_3/Task1.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+
+
+
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java:16]: Please remove extra lines

File: src/main/java/hw_3/Task1.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package hw_3;
+
+import java.util.Arrays;
+
+public class Task1 {
+    public static void main(String[] args) {
+            //Виведіть у консоль один рядок з числами з масиву (які ми створювали в 1-му завданні), розділеними “;”
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: There should be no Latin words

File: src/main/java/hw_3/Task1.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package hw_3;
+
+import java.util.Arrays;
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: Unused imports should be removed

File: src/main/java/hw_3/Task2.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
+package hw_3;
+
+public class Task2 {
+    public static void main(String[] args) {
+        //Створіть змінну із рядком “Мама мила раму”. Створіть масив зі всіх слів, які зустрічаються у цьому рядку.
+        // (Погляньте на методи рядка та подумайте, який метод можна використовувати)
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task2.java]: There should be no Latin words

File: src/main/java/hw_3/Task3.java [hunk 6]
```diff
@@ -0,0 +1,12 @@
+package hw_3;
+
+public class Task3 {
+    public static void main(String[] args) {
+        //Створіть рядок "Я хочу перенесення рядків."
+        // Замініть всі символи пробілу в рядку на символи перенесення рядків та виведіть у консоль.
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task3.java]: There should be no Latin words

File: src/main/java/hw_3/Task4.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
+package hw_3;
+import java.util.Arrays;
+public class Task4 {
+    public static void main(String[] args) {
+       //Створіть масив із цілих чисел і помістіть відразу 10 значень. Пройдіться ними і змініть значення.
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task4.java]: There should be no Latin words

File: src/main/java/hw_3/Task4.java [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+        int[] arr = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+
+        System.out.println(Arrays.toString(arr));
+
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = arr[i] * 2;  // Наприклад, множимо кожне значення на 2
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task4.java]: There should be no Latin words

File: src/main/java/hw_3/Task5.java [hunk 9]
```diff
@@ -0,0 +1,12 @@
+package hw_3;
+
+import java.util.Arrays;
+
+public class Task5 {
+    public static void main(String[] args) {
+        //Створіть масив із дробових чисел (5 елементів).
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task5.java]: There should be no Latin words

File: src/main/java/hw_3/Task6.java [hunk 10]
```diff
@@ -0,0 +1,18 @@
+package hw_3;
+
+import java.util.Arrays;
+
+public class Task6 {
+    public static void main(String[] args) {
+        //Створіть 3 змінних таких типів: byte, int, long. Створіть масив типу int. Спробуйте туди покласти змінні.
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task6.java]: There should be no Latin words

File: src/main/java/hw_3/Task7.java [hunk 11]
```diff
@@ -0,0 +1,13 @@
+package hw_3;
+
+import java.util.Arrays;
+
+public class Task7 {
+    public static void main(String[] args) {
+        //Створіть будь-який двовимірний масив, заповніть будь-якими даними та виведіть у консоль
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task7.java]: There should be no Latin words

File: src/main/java/hw_3/Task8.java [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+public class Task8 {
+    public static void main(String[] args) {
+        /*Створіть рядок "Hello world!"
+                - Виведіть перший символ
+                - Виведіть останній символ
+                - Створіть масив символів, покладіть туди усі символи слова world
+                - Вивести масив у консоль
+                - Вивести в консоль передостанній елемент масиву, не використовуючи прямий доступ
+                за індексом (arr[3] – не згодиться)
+         */
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task8.java]: There should be no Latin words

File: src/main/java/hw_3/Task2.java [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+        String sentence = "Mother was washing the window";
+        String[] words = sentence.split(" ");
+        for (int i = 0; i < words.length; i++) {
+            if (i == words.length - 1) {
+                System.out.println(words[i]);
+            } else {
+                System.out.println(words[i] + " ");
+            }
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task2.java]: Why do we need this if else construction?
Why just not System.out.println(words[i]);   ?

File: src/main/java/hw_3/Task6.java [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+        byte a = 12;
+        int b = 210;
+        long c = 3113;
+
+       // int[] = {a, b, c} error, Java does not allow mixing primitive types in arrays directly
+
+        Object[] differentTypes = {a, b, c};
+
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task6.java]: You need to create an int[] array and put all your above values into it. Try to implement it.

File: src/main/java/hw_3/Task8.java [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+
+        String line = "Hello world!";
+        char firstChar = line.charAt(0);
+        char lastChar = line.charAt(line.length() - 1);
+
+        System.out.println("Перший символ: " + firstChar);
+        System.out.println("Останній символ: " + lastChar);
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task8.java]: There should be no Latin words. Please fix in all places

File: src/main/java/hw_3/Task8.java [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+
+        char[] worldArray = {'w', 'o', 'r', 'l', 'd'};
+        char a = worldArray[worldArray.length - 2];
+        System.out.println("Останній елемент масиву: " + a);
+       // System.out.println(Arrays.toString([worldArray.length - 1]);
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task8.java]: THis commented code can be removed?

File: src/main/java/hw_3/Task8.java [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+         */
+
+        String line = "Hello world!";
+        char firstChar = line.charAt(0);
+        char lastChar = line.charAt(line.length() - 1);
+
+        System.out.println("Перший символ: " + firstChar);
+        System.out.println("Останній символ: " + lastChar);
+
+        char[] worldArray = {'w', 'o', 'r', 'l', 'd'};
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task8.java]: 1) You need a separate variable with "world" string value
2) You need to create a char[] and put your string there as a list of chars
3) You need to display you array
4) You need to display second last element for array

image

File: src/main/java/hw_3/Task1.java (hunk 1 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task1.java]: fixed

File: src/main/java/hw_3/Task1.java (hunk 2 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task1.java:16]: fixed

File: src/main/java/hw_3/Task1.java (hunk 3 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task1.java]: fixed

File: src/main/java/hw_3/Task1.java (hunk 4 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task1.java]: fixed

File: src/main/java/hw_3/Task2.java (hunk 5 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task2.java]: fixed

File: src/main/java/hw_3/Task3.java (hunk 6 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task3.java]: fixed

File: src/main/java/hw_3/Task4.java (hunk 7 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task4.java]: fixed

File: src/main/java/hw_3/Task4.java (hunk 8 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task4.java]: fixed

File: src/main/java/hw_3/Task5.java (hunk 9 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task5.java]: fixed

File: src/main/java/hw_3/Task6.java (hunk 10 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task6.java]: fixed

File: src/main/java/hw_3/Task7.java (hunk 11 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task7.java]: fixed

File: src/main/java/hw_3/Task8.java (hunk 12 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task8.java]: fixed

File: src/main/java/hw_3/Task2.java (hunk 13 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task2.java]: fixed

File: src/main/java/hw_3/Task6.java (hunk 14 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task6.java]: fixed

File: src/main/java/hw_3/Task8.java (hunk 15 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task8.java]: fixed

File: src/main/java/hw_3/Task8.java (hunk 16 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task8.java]: fixed

File: src/main/java/hw_3/Task8.java (hunk 17 above)
Comment (AntTurieva) [reply src/main/java/hw_3/Task8.java]: fixed

**Pull request merged**